sdks/go/pkg/beam/testing/passert/passert.go

```diff
@@ -32,29 +32,6 @@ import (
 //go:generate starcgen --package=passert --identifiers=diffFn,failFn,failKVFn,failGBKFn,hashFn,sumFn
```

Comment:
do i need to add something here (or add a similar stanza to the new equals.go file)?

Reply:
failIfBadEntries should be added, at least, and technically, `go generate` should be run to update the .shim.go file. 

Given there's work to be done in the code generator in the next month or so, and that this is just for testing, I'm less worried about not having the shim file updated. Having the identifier added though is important to avoid losing it later.

Reply:
I added the thing and updated the shim file as well.